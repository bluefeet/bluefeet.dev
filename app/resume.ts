import zResume from "./zResume";
import rawResume from "@/public/resume.json";
import { z } from "zod";

/**
 * The full, from JSON, {@link rawResume} cast as a {@link Resume}.
 */
export const resume = zResume.parse(rawResume);

export type Resume = z.infer<typeof zResume>;

const zExperience = zResume.shape.experiences.unwrap().element;
export type Experience = z.infer<typeof zExperience>;

const zRecommendation = zResume.shape.recommendations.unwrap().element;
export type Recommendation = z.infer<typeof zRecommendation>;
