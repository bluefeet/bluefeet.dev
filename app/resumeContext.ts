import { createContext, useContext } from 'react'
import { Resume } from '@/types/Resume'

/**
 * The default resume if none is set in the context. Exported so that tests can
 * use this as a foundation for building test resumes.
 */
export const emptyResume: Resume = {
  $schema: 'resume.schema.json',
}

const resumeContext = createContext<Resume>(emptyResume)

/**
 * Used to avoid prop drilling since the resume is accessed everywhere.
 */
export const ResumeProvider = resumeContext.Provider

/**
 * Provides access to the current resume as set by the {@link ResumeProvider}.
 */
export const useResume = () => useContext(resumeContext)
