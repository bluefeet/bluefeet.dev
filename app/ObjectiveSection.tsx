import { Details } from "./Details";
import { List } from "./List";
import { ListItem } from "./ListItem";
import { useResume } from "./resumeContext";
import {
  BuildingOffice2Icon,
  CalendarDaysIcon,
  Cog6ToothIcon,
  PhotoIcon,
  RocketLaunchIcon,
} from "@heroicons/react/24/solid";
import { format as formatDate } from "date-fns/format";
import { isPast as isDateInPast } from "date-fns/isPast";
import { parseISO as parseISODate } from "date-fns/parseISO";
import upperFirst from "lodash/upperFirst";

const startDateFormat = "PPP";

const Overview = () => {
  const resume = useResume();
  if (!resume.objective?.overview) return <></>;

  return (
    <>
      <ListItem Icon={PhotoIcon}>
        Dream Job Traits
        <Details>{resume.objective.overview}</Details>
      </ListItem>
    </>
  );
};

const StartDate = () => {
  const resume = useResume();
  if (!resume.objective) return <></>;

  if ((resume.objective.intention || "passive") === "passive") return <></>;

  let startDate = resume.objective.startDate
    ? parseISODate(resume.objective.startDate)
    : null;

  // Don't display if in the past.
  if (startDate && isDateInPast(startDate)) startDate = null;

  return (
    <>
      <ListItem Icon={RocketLaunchIcon}>
        {resume.objective.intention === "casual"
          ? "Open to new opportunities"
          : "Actively searching for an opportunity"}
        {
          <Details>
            Available{" "}
            {startDate ? (
              <>starting on {formatDate(startDate, startDateFormat)}</>
            ) : (
              <>immediately</>
            )}
          </Details>
        }
      </ListItem>
    </>
  );
};

const Roles = () => {
  const resume = useResume();
  if (!resume.objective?.roles?.length) return <></>;

  return (
    <>
      <ListItem Icon={Cog6ToothIcon}>
        {resume.objective.roles.length > 1 ? "These roles" : "This role"} would
        be a great match
        <Details>
          {resume.objective.roles.map((role) => (
            <div key={role}>{role}</div>
          ))}
        </Details>
      </ListItem>
    </>
  );
};

const WorkModes = () => {
  const resume = useResume();
  if (!resume.objective?.workModes?.length) return <></>;

  return (
    <>
      <ListItem Icon={BuildingOffice2Icon}>
        In{" "}
        {resume.objective.workModes.length > 1
          ? "any of these capacities"
          : "this capacity"}
        <Details>
          {resume.objective.workModes
            .map((workMode) => upperFirst(workMode))
            .join(", ")}
        </Details>
      </ListItem>
    </>
  );
};

const EmploymentTypes = () => {
  const resume = useResume();
  if (!resume.objective?.employmentTypes?.length) return <></>;

  return (
    <>
      <ListItem Icon={CalendarDaysIcon}>
        With{" "}
        {resume.objective.employmentTypes.length > 1
          ? "one of these commitments"
          : "this commitment"}
        <Details>
          {resume.objective.employmentTypes
            .map((employmentType) => upperFirst(employmentType))
            .join(", ")}
        </Details>
      </ListItem>
    </>
  );
};

export const ObjectiveSection = ({
  className = "",
}: {
  className?: string;
}) => (
  <>
    <section className={className}>
      <List>
        <Overview />
        <StartDate />
        <Roles />
        <EmploymentTypes />
        <WorkModes />
      </List>
    </section>
  </>
);
