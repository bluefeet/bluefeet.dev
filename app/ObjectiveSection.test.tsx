import { ObjectiveSection } from "./ObjectiveSection";
import { Resume } from "./resume";
import { emptyResume, ResumeProvider } from "./resumeContext";
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

const testResume = (resume: Partial<Resume>) => {
  render(
    <ResumeProvider value={{ ...emptyResume, ...resume }}>
      <ObjectiveSection />
    </ResumeProvider>,
  );
  expect(document.body).toMatchSnapshot();
};

describe("ObjectiveSection", () => {
  it("empty resume", () => {
    testResume({});
  });

  it("overview", () => {
    testResume({ objective: { overview: "test-overview" } });
  });

  describe("startDate", () => {
    it("objective with no startDate", () => {
      testResume({ objective: { intention: "casual" } });
    });

    it("casual startDate", () => {
      testResume({
        objective: { intention: "casual", startDate: "2500-11-04" },
      });
    });

    it("active startDate", () => {
      testResume({
        objective: { intention: "active", startDate: "2500-11-04" },
      });
    });

    it("historic startDate", () => {
      testResume({
        objective: { intention: "casual", startDate: "1905-08-17" },
      });
    });
  });

  describe("roles, workModes, and employmentTypes", () => {
    it("single", () => {
      testResume({
        objective: {
          roles: ["foo"],
          workModes: ["on-site"],
          employmentTypes: ["full-time"],
        },
      });
    });

    it("multiple", () => {
      testResume({
        objective: {
          roles: ["foo", "bar"],
          workModes: ["hybrid", "remote"],
          employmentTypes: ["freelance", "internship"],
        },
      });
    });
  });
});
