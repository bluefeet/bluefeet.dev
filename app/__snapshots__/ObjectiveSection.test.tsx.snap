// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ObjectiveSection empty resume 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      />
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection overview 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      >
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M1.5 6a2.25 2.25 0 0 1 2.25-2.25h16.5A2.25 2.25 0 0 1 22.5 6v12a2.25 2.25 0 0 1-2.25 2.25H3.75A2.25 2.25 0 0 1 1.5 18V6ZM3 16.06V18c0 .414.336.75.75.75h16.5A.75.75 0 0 0 21 18v-1.94l-2.69-2.689a1.5 1.5 0 0 0-2.12 0l-.88.879.97.97a.75.75 0 1 1-1.06 1.06l-5.16-5.159a1.5 1.5 0 0 0-2.12 0L3 16.061Zm10.125-7.81a1.125 1.125 0 1 1 2.25 0 1.125 1.125 0 0 1-2.25 0Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          Dream Job Traits
          <div
            class="text-zinc-300 text-sm "
          >
            test-overview
          </div>
        </div>
      </div>
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection roles, workModes, and employmentTypes multiple 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      >
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M11.078 2.25c-.917 0-1.699.663-1.85 1.567L9.05 4.889c-.02.12-.115.26-.297.348a7.493 7.493 0 0 0-.986.57c-.166.115-.334.126-.45.083L6.3 5.508a1.875 1.875 0 0 0-2.282.819l-.922 1.597a1.875 1.875 0 0 0 .432 2.385l.84.692c.095.078.17.229.154.43a7.598 7.598 0 0 0 0 1.139c.015.2-.059.352-.153.43l-.841.692a1.875 1.875 0 0 0-.432 2.385l.922 1.597a1.875 1.875 0 0 0 2.282.818l1.019-.382c.115-.043.283-.031.45.082.312.214.641.405.985.57.182.088.277.228.297.35l.178 1.071c.151.904.933 1.567 1.85 1.567h1.844c.916 0 1.699-.663 1.85-1.567l.178-1.072c.02-.12.114-.26.297-.349.344-.165.673-.356.985-.57.167-.114.335-.125.45-.082l1.02.382a1.875 1.875 0 0 0 2.28-.819l.923-1.597a1.875 1.875 0 0 0-.432-2.385l-.84-.692c-.095-.078-.17-.229-.154-.43a7.614 7.614 0 0 0 0-1.139c-.016-.2.059-.352.153-.43l.84-.692c.708-.582.891-1.59.433-2.385l-.922-1.597a1.875 1.875 0 0 0-2.282-.818l-1.02.382c-.114.043-.282.031-.449-.083a7.49 7.49 0 0 0-.985-.57c-.183-.087-.277-.227-.297-.348l-.179-1.072a1.875 1.875 0 0 0-1.85-1.567h-1.843ZM12 15.75a3.75 3.75 0 1 0 0-7.5 3.75 3.75 0 0 0 0 7.5Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          These roles
           would be a great match
          <div
            class="text-zinc-300 text-sm "
          >
            <div>
              foo
            </div>
            <div>
              bar
            </div>
          </div>
        </div>
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12.75 12.75a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM7.5 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM8.25 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM9.75 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM10.5 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM12 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM12.75 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM14.25 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM15 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM16.5 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM15 12.75a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM16.5 13.5a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5Z"
            />
            <path
              clip-rule="evenodd"
              d="M6.75 2.25A.75.75 0 0 1 7.5 3v1.5h9V3A.75.75 0 0 1 18 3v1.5h.75a3 3 0 0 1 3 3v11.25a3 3 0 0 1-3 3H5.25a3 3 0 0 1-3-3V7.5a3 3 0 0 1 3-3H6V3a.75.75 0 0 1 .75-.75Zm13.5 9a1.5 1.5 0 0 0-1.5-1.5H5.25a1.5 1.5 0 0 0-1.5 1.5v7.5a1.5 1.5 0 0 0 1.5 1.5h13.5a1.5 1.5 0 0 0 1.5-1.5v-7.5Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          With
           
          one of these commitments
          <div
            class="text-zinc-300 text-sm "
          >
            Freelance, Internship
          </div>
        </div>
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M3 2.25a.75.75 0 0 0 0 1.5v16.5h-.75a.75.75 0 0 0 0 1.5H15v-18a.75.75 0 0 0 0-1.5H3ZM6.75 19.5v-2.25a.75.75 0 0 1 .75-.75h3a.75.75 0 0 1 .75.75v2.25a.75.75 0 0 1-.75.75h-3a.75.75 0 0 1-.75-.75ZM6 6.75A.75.75 0 0 1 6.75 6h.75a.75.75 0 0 1 0 1.5h-.75A.75.75 0 0 1 6 6.75ZM6.75 9a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75ZM6 12.75a.75.75 0 0 1 .75-.75h.75a.75.75 0 0 1 0 1.5h-.75a.75.75 0 0 1-.75-.75ZM10.5 6a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75Zm-.75 3.75A.75.75 0 0 1 10.5 9h.75a.75.75 0 0 1 0 1.5h-.75a.75.75 0 0 1-.75-.75ZM10.5 12a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75ZM16.5 6.75v15h5.25a.75.75 0 0 0 0-1.5H21v-12a.75.75 0 0 0 0-1.5h-4.5Zm1.5 4.5a.75.75 0 0 1 .75-.75h.008a.75.75 0 0 1 .75.75v.008a.75.75 0 0 1-.75.75h-.008a.75.75 0 0 1-.75-.75v-.008Zm.75 2.25a.75.75 0 0 0-.75.75v.008c0 .414.336.75.75.75h.008a.75.75 0 0 0 .75-.75v-.008a.75.75 0 0 0-.75-.75h-.008ZM18 17.25a.75.75 0 0 1 .75-.75h.008a.75.75 0 0 1 .75.75v.008a.75.75 0 0 1-.75.75h-.008a.75.75 0 0 1-.75-.75v-.008Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          In
           
          any of these capacities
          <div
            class="text-zinc-300 text-sm "
          >
            Hybrid, Remote
          </div>
        </div>
      </div>
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection roles, workModes, and employmentTypes single 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      >
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M11.078 2.25c-.917 0-1.699.663-1.85 1.567L9.05 4.889c-.02.12-.115.26-.297.348a7.493 7.493 0 0 0-.986.57c-.166.115-.334.126-.45.083L6.3 5.508a1.875 1.875 0 0 0-2.282.819l-.922 1.597a1.875 1.875 0 0 0 .432 2.385l.84.692c.095.078.17.229.154.43a7.598 7.598 0 0 0 0 1.139c.015.2-.059.352-.153.43l-.841.692a1.875 1.875 0 0 0-.432 2.385l.922 1.597a1.875 1.875 0 0 0 2.282.818l1.019-.382c.115-.043.283-.031.45.082.312.214.641.405.985.57.182.088.277.228.297.35l.178 1.071c.151.904.933 1.567 1.85 1.567h1.844c.916 0 1.699-.663 1.85-1.567l.178-1.072c.02-.12.114-.26.297-.349.344-.165.673-.356.985-.57.167-.114.335-.125.45-.082l1.02.382a1.875 1.875 0 0 0 2.28-.819l.923-1.597a1.875 1.875 0 0 0-.432-2.385l-.84-.692c-.095-.078-.17-.229-.154-.43a7.614 7.614 0 0 0 0-1.139c-.016-.2.059-.352.153-.43l.84-.692c.708-.582.891-1.59.433-2.385l-.922-1.597a1.875 1.875 0 0 0-2.282-.818l-1.02.382c-.114.043-.282.031-.449-.083a7.49 7.49 0 0 0-.985-.57c-.183-.087-.277-.227-.297-.348l-.179-1.072a1.875 1.875 0 0 0-1.85-1.567h-1.843ZM12 15.75a3.75 3.75 0 1 0 0-7.5 3.75 3.75 0 0 0 0 7.5Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          This role
           would be a great match
          <div
            class="text-zinc-300 text-sm "
          >
            <div>
              foo
            </div>
          </div>
        </div>
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M12.75 12.75a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM7.5 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM8.25 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM9.75 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM10.5 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM12 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM12.75 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM14.25 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM15 17.25a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM16.5 15.75a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5ZM15 12.75a.75.75 0 1 1-1.5 0 .75.75 0 0 1 1.5 0ZM16.5 13.5a.75.75 0 1 0 0-1.5.75.75 0 0 0 0 1.5Z"
            />
            <path
              clip-rule="evenodd"
              d="M6.75 2.25A.75.75 0 0 1 7.5 3v1.5h9V3A.75.75 0 0 1 18 3v1.5h.75a3 3 0 0 1 3 3v11.25a3 3 0 0 1-3 3H5.25a3 3 0 0 1-3-3V7.5a3 3 0 0 1 3-3H6V3a.75.75 0 0 1 .75-.75Zm13.5 9a1.5 1.5 0 0 0-1.5-1.5H5.25a1.5 1.5 0 0 0-1.5 1.5v7.5a1.5 1.5 0 0 0 1.5 1.5h13.5a1.5 1.5 0 0 0 1.5-1.5v-7.5Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          With
           
          this commitment
          <div
            class="text-zinc-300 text-sm "
          >
            Full-time
          </div>
        </div>
        <div
          class="p-1 pl-8 pr-8 flex place-items-center"
        >
          <svg
            aria-hidden="true"
            class="w-6 text-amber-400 print:text-sky-600"
            data-slot="icon"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M3 2.25a.75.75 0 0 0 0 1.5v16.5h-.75a.75.75 0 0 0 0 1.5H15v-18a.75.75 0 0 0 0-1.5H3ZM6.75 19.5v-2.25a.75.75 0 0 1 .75-.75h3a.75.75 0 0 1 .75.75v2.25a.75.75 0 0 1-.75.75h-3a.75.75 0 0 1-.75-.75ZM6 6.75A.75.75 0 0 1 6.75 6h.75a.75.75 0 0 1 0 1.5h-.75A.75.75 0 0 1 6 6.75ZM6.75 9a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75ZM6 12.75a.75.75 0 0 1 .75-.75h.75a.75.75 0 0 1 0 1.5h-.75a.75.75 0 0 1-.75-.75ZM10.5 6a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75Zm-.75 3.75A.75.75 0 0 1 10.5 9h.75a.75.75 0 0 1 0 1.5h-.75a.75.75 0 0 1-.75-.75ZM10.5 12a.75.75 0 0 0 0 1.5h.75a.75.75 0 0 0 0-1.5h-.75ZM16.5 6.75v15h5.25a.75.75 0 0 0 0-1.5H21v-12a.75.75 0 0 0 0-1.5h-4.5Zm1.5 4.5a.75.75 0 0 1 .75-.75h.008a.75.75 0 0 1 .75.75v.008a.75.75 0 0 1-.75.75h-.008a.75.75 0 0 1-.75-.75v-.008Zm.75 2.25a.75.75 0 0 0-.75.75v.008c0 .414.336.75.75.75h.008a.75.75 0 0 0 .75-.75v-.008a.75.75 0 0 0-.75-.75h-.008ZM18 17.25a.75.75 0 0 1 .75-.75h.008a.75.75 0 0 1 .75.75v.008a.75.75 0 0 1-.75.75h-.008a.75.75 0 0 1-.75-.75v-.008Z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
        <div
          class="pt-2 pb-2"
        >
          In
           
          this capacity
          <div
            class="text-zinc-300 text-sm "
          >
            On-site
          </div>
        </div>
      </div>
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection startDate casual startDate 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      />
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection startDate historic startDate 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      />
    </section>
  </div>
</body>
`;

exports[`ObjectiveSection startDate objective with no startDate 1`] = `
<body>
  <div>
    <section
      class=""
    >
      <div
        class="grid grid-cols-[min-content_auto]"
      />
    </section>
  </div>
</body>
`;
